docs/USAGE.md

```diff
@@ -61,20 +53,23 @@ In your `.env` you should have a `STRIPE_SECRET_KEY` and `STRIPE_PLATFORM_CLIENT
 
 ## Generating Documentation
 
-The Code Corps Ember application uses [YUIDoc](http://yui.github.io/yuidoc/) for documentation. When contributing to the documentation please follow the [YUIDoc syntax](http://yui.github.io/yuidoc/syntax/index.html) and our [style guide](STYLEGUIDE.md) for application specific documenation standards.
+The Code Corps Ember application uses [YUIDoc](http://yui.github.io/yuidoc/) for documentation. 
+
+When contributing to the documentation please follow our [style guide](STYLEGUIDE.md) and [YUIDoc's syntax guidelines](http://yui.github.io/yuidoc/syntax/index.html) for application specific documenation standards.
```

Comment:
Ordering was changed but now the second half of the sentence doesn't make sense.